src/sections/Meshery/Meshery-integrations/Integration.style.js

```diff
@@ -29,6 +29,12 @@ export const HoneycombGrid = styled.div`
     color: #1e2117;
     font-size: 0.875rem;
     cursor: pointer;
+
+    &:hover {
+      opacity: 0.8;
+      transition: all .2s ease-in-out;
```

Comment:
The `transition` property should be applied to the default CSS and not the `:hover` selector, else it won't be effective.

Reply:
Preferred value for `box-shadow` `rgb(60 73 79) 0px 0px 6px -2px`.
And, update the time to `0.1s`

Reply:
okay I am updating the changes